### PR TITLE
Fix flaky test for multi profiles and alerts for props

### PR DESCRIPTION
**Pull request by pa-lem**
* Fixes test to wait after each creation
* Fixes alerts to correctly use toast props

### CODE REVIEW

File: frontend/app/src/Root.tsx [hunk 1]
```diff
@@ -21,9 +21,9 @@ export const Root = ({ children }: { children?: ReactNode }) => {
     try {
       return fetchUrl(CONFIG.CONFIG_URL);
     } catch (err) {
-      toast(() => (
+      toast(
```
Comment (bilalabbad) [inline frontend/app/src/Root.tsx:24]: all these changes on `toast` makes an error on browser console + introduce flakiness on toast

Comment (bilalabbad) [reply frontend/app/src/Root.tsx:24]: if you want to pass params, prefer making it explicit via props passing

Comment (pa-lem) [reply frontend/app/src/Root.tsx:24]: the warnings are addressed and I don't see where the flakiness can come from, what I can see now is that we don't wait correctly when accessing a page and before creating an object

**Review by bilalabbad — APPROVED** [review]
lgtm, try to reload the CI several times to check if there is still some flakiness remaining

**Pull request merged**